WalletWasabi/Crypto/OwnershipProof.cs

```diff
@@ -97,21 +99,21 @@ public void ReadWrite(BitcoinStream bitcoinStream)
 				}
 			}
 
-			bitcoinStream.ReadWrite(ref flags);
-			bitcoinStream.ReadWrite(ref ownershipIdentifiers);
+			bitcoinStream.ReadWrite(ref _flags);
+			bitcoinStream.ReadWrite(ref _ownershipIdentifiers);
 		}
 
 		public uint256 SignatureHash(Script scriptPubKey, byte[] commitmentData) =>
-			new uint256(Hashes.SHA256(this.ToBytes().Concat(ProofFooter(scriptPubKey, commitmentData)).ToArray()));
+			new(Hashes.SHA256(this.ToBytes().Concat(ProofFooter(scriptPubKey, commitmentData)).ToArray()));
```

Comment:
IDK. There is clearly places where `new()` has a lot of sense, for example en places where there is no other choice, in property initialization and perhaps a few others but this is imo no one of those. I mean, here is easy to see that it creates a new `uint256` because that's what the method signature says it returns but that is only because this method is implemented with a one-line expression body.

Imagine you get to the end of a 40LoC method and you see `return new(x));` what is it returning!? Well, in order to know that you need to scroll up 40LoC and check the method signature.

I am not requesting changes here but just to think every time we replace a type.